### PR TITLE
Andres neo4j

### PR DESCRIPTION
**Pull request by andreshyer**
Branch to interact with neo4j with python, using the bulk chemical data as a starting place.

I did not touch any of the previous scripts, so this pull request should not break any of our code. But, please check and make sure that nothing important broke.

### CODE REVIEW

File: Neo4j/BulkChem/ChemNeo4j.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        if file is not None:  # Get data from bulkchem csv
+            bulk_data = pd.read_csv(file)
+            self.bulk_dicts = bulk_data.to_dict('records')
+
+        print('Comparing Bulk Chem Data with relationship {0}'.format(self.protocol))
+        self.compare_molecules()
+
+
+insert_bulk_chem_molecules('BulkChemData.csv')
+create_relationships('TESTING', retrieve_data_from_neo4j=True)
```
Comment (qle2) [inline Neo4j/BulkChem/ChemNeo4j.py]: Hey @andreshyer, I notice that there are multiple versions of ChemNeo4j.py. Do you need all of them for your work?
Also, can you please add doc strings for each of your functions to explain its inputs and outputs?

**Comment by Dlux804** [comment]
@andreshyer What is the status of this PR?  

Are all of Kaitlins CSV files necessary?  Is there a more effecient way to provide that info?  Please do not spend too much time on her data -- it's hairy and a priority for our team.

**Comment by andreshyer** [comment]
I believe I am still waiting for approve before being able to merge the
pull request. The files are definitely not necessary, I have it there if I
ever have free time and do not know what else to do, which seems unlikely.
The files will be removed, and I can update the pull request. If I ever get
around to adding her files to a GB, I can update the repo with the script
to do so. If we still wanted the files on a database, we can move them to
dropbox.

File: Neo4j/BulkChem/ChemNeo4j.py (hunk 1 above)
Comment (andreshyer) [reply Neo4j/BulkChem/ChemNeo4j.py]: I can add doc strings. No the other files are not needed, but there are much easier to read and understand in my option. But you would rather have large doc string explaining the code, I can do that instead

**Comment by andreshyer** [comment]
> 
> 
> @andreshyer What is the status of this PR?
> 
> Are all of Kaitlins CSV files necessary? Is there a more effecient way to provide that info? Please do not spend too much time on her data -- it's hairy and not a priority for our team.

I believe I am still waiting for approve before being able to merge the
pull request. The files are definitely not necessary, I have it there if I
ever have free time and do not know what else to do, which seems unlikely.
The files will be removed, and I can update the pull request. If I ever get
around to adding her files to a GB, I can update the repo with the script
to do so. If we still wanted the files on a database, we can move them to
dropbox.

File: Neo4j/BulkChem/ChemNeo4j.py (hunk 1 above)
Comment (Dlux804) [reply Neo4j/BulkChem/ChemNeo4j.py]: Have one version of the file and add in the explanation necessary to make it understandable.  That can be via comments and doc strings.

**Comment by Dlux804** [comment]
@andreshyer  Andres, please update the code with the documentation/explanation requested by @qle2.  I agree with your suggestion of moving Kaitlin's files from here and making a folder on dropbox for them which we can access later if the need arises.  

Also please add a README.md to the Neo4j/ or BulkChem/ directory explaining at a high level what is going on and how to get started working with the code (i.e any particular installation instructions, organization of the files, etc.)

Once this is done, please update this  PR and notify @qle2 to review your branch.

**Comment by andreshyer** [comment]
I am closing this pull request, as branch needs further work before merging

**Comment by andreshyer** [comment]
> 
> 
> @andreshyer Andres, please update the code with the documentation/explanation requested by @qle2. I agree with your suggestion of moving Kaitlin's files from here and making a folder on dropbox for them which we can access later if the need arises.
> 
> Also please add a README.md to the Neo4j/ or BulkChem/ directory explaining at a high level what is going on and how to get started working with the code (i.e any particular installation instructions, organization of the files, etc.)
> 
> Once this is done, please update this PR and notify @qle2 to review your branch.

Will do, will update when work is complete

**Pull request closed without merging**